content/documentation/staging/tutorial/06-secure/01-traffic-policies.adoc

```diff
@@ -14,7 +14,9 @@ In this tutorial we will show how Kiali can use telemetry information to create
 
 Istio telemetry aggregates the ServiceAccount information used in the workloads communication. This information can be used to define authorization policies that deny and allow actions on future live traffic communication status.
 
-This step will show how we can define authorization policies for the *travel-agency* namespace in the Travel Demo application, for all existing traffic in a given time period.
+Besides authorization policies, sidecars can be created to limit the hosts a given workload can communicate with. This helps to better control traffic and also to reduce the https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#cpu-and-memory[memory footprint, window="_blank"] of the proxies.
```

Comment:
This is style and probably @jshaughn may have better suggestions than me on this.

But I'd start the phrase like:

"Additionally, Istio sidecars can be created to ..."